packages/browser-wallet/src/shared/storage/transactions.ts

```diff
@@ -0,0 +1,22 @@
+import { BrowserWalletAccountTransaction } from '@popup/shared/utils/transaction-history-types';
+import { makeStorageAccessor } from './access';
+import { ChromeStorageKey, StoredBrowserWalletTransaction } from './types';
+
+export const pendingTransactions = makeStorageAccessor<StoredBrowserWalletTransaction[]>(
+    'local',
```

Comment:
We might want to consider just putting this in `session` storage. If the user closes their browser, then the transaction will probably finalize before they open their wallet again?

Reply:
Sure, no need to run unnecessary cleanup.